dspace/config/modules/external-providers.cfg

```diff
@@ -25,4 +25,25 @@ vufind.url.search = https://vufind.org/advanced_demo/api/v1/search
 
 scielo.url = https://search.scielo.org/?output=ris&q=
 
-#################################################################
\ No newline at end of file
+#################################################################
+#----------------------   ADS   --------------------------------#
```

Comment:
Let's make sure to say what ADS is in this section header.  So, this is the NASA Astrophysics Data  System (ADS)